client/src/theme.js

```diff
@@ -3,11 +3,15 @@ import styled from "styled-components";
 export const Theme = {
   color: {
```

Comment:
I would try have the theme colours come from the same place. Just so we don't have the duplication in src/styled and src/theme

Reply:
The theme file isn't being used anymore so I've removed it from the repo